libs/wingsdk/src/target-tf-aws/table.ts

```diff
@@ -0,0 +1,110 @@
+import { DynamodbTable } from "@cdktf/provider-aws/lib/dynamodb-table";
+import { Construct } from "constructs";
+import { Function } from "./function";
+import * as cloud from "../cloud";
+import * as core from "../core";
+import { NameOptions, ResourceNames } from "../utils/resource-names";
+
+/**
+ * Counter (Table) names must be between 3 and 255 characters.
+ * You can use alphanumeric characters, dot (.), dash (-), and underscores (_).
```

Comment:
I know we dont do this yet for other resources but perhaps we should start adding links to reference where the naming convention rules came from in documentation. Since these are target specific it might just make it easier for future development.